charts/edge/templates/NOTES.txt

```diff
@@ -1,7 +1,8 @@
-{{- if not .Values.mezmoApiAccessToken -}}
-#####################################################################
-###  ERROR: Please specify an access token `mezmoApiAccessToken`  ###
-#####################################################################
+{{- if and (not .Values.mezmoApiAccessToken) (not .Values.mezmoApiAccessSecret) -}}
+######################################################################
+###  ERROR: Please specify an access token (`mezmoApiAccessToken`) ###
+###    or a secret containing a token (`mezmoApiAccessSecret`)     ###
+######################################################################
```

Comment:
Clarified a bit to indicate which one takes precedence rather than using mutually exclusive. Also updated to block the installation instead of just showing an error